src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,71 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
+
+    public int calculate(String inputString){
+        for(String splitData : inputString.split(" ")){
+                A(splitData);
+        }
+        return numbers.pop();
+    }
+
+    private void A(String splitData) {
+        switch (sign.getValue(splitData)){
+            case ADDITION:
+                numbers.push(addition());
+                break;
+            case SUBTRACTION:
+                numbers.push(subtraction());
+                break;
+            case MULTIPLICATION:
+                numbers.push(multiplication());
+                break;
+            case DIVISION:
+                numbers.push(division());
+                break;
+            case NUMBER:
+                numbers.push(Integer.parseInt(splitData));
+                break;
+        }
+    }
+
+    private final int addition(){
+        int result;
+        final int secondNumber = numbers.pop();
+        final int firstNumber = numbers.pop();
+
+        result = firstNumber + secondNumber;
+        return result;
```

Comment:
you can make code here a little shorter. `return first + second` . To make code shorter is not always means the best solution but here when it's few lines it's definitely better